www/book.css

```diff
@@ -13,7 +28,11 @@ body {
 }
 
 pre, code { hyphens: none; -webkit-hyphens: none; }
-pre { font-size: 18px; overflow: auto; padding-left: 2ex; }
+pre {
+    font-size: 18px; overflow: auto; padding-left: 2ex;
+    background-color: #ddd; /* For WBE browser only */
```

Comment:
Does the WBE browser properly ignore comments?

Reply:
No, or, uhh, sortof. It doesn't have handling for comments, but when it sees a comment it treats it as a syntax error which causes it to eat everything up to the next semicolon. So this comment sort of deletes the next property (`background: transparent`). This doesn't change how our browser behaves because it doesn't support `background` anyway.